akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -37,6 +40,23 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
     hctx.map(_.asInstanceOf[akka.http.scaladsl.HttpsContext])
 
   /**
+   * Constructs a server layer stage using the configured default [[ServerSettings]]. The returned [[BidiFlow]] isn't
+   * reusable and can only be materialized once.
+   */
+  def serverLayer(materializer: Materializer): BidiFlow[HttpResponse, SslTlsOutbound, SslTlsInbound, HttpRequest, Unit] =
+    adaptServerLayer(delegate.serverLayer()(materializer))
+
+  /**
+   * Constructs a server layer stage using the given [[ServerSettings]]. The returned [[BidiFlow]] isn't reusable and
+   * can only be materialized once.
+   */
+  def serverLayer(settings: ServerSettings,
+                  remoteAddress: Option[InetSocketAddress],
+                  log: LoggingAdapter,
+                  materializer: Materializer): BidiFlow[HttpResponse, SslTlsOutbound, SslTlsInbound, HttpRequest, Unit] =
```

Comment:
Yes, I agree. That's only a tiny bit of stupid work more us for the benefit of all those Java users out there.